build-tests/heft-sass-test/config/sass.json

```diff
@@ -0,0 +1,3 @@
+{
+  "cssOutputFolder": "lib"
```

Comment:
Do you think this should be an array? As long as we only intend to run the bundler in one output directory, then having a single output folder is all we need. And I can't think of any good reason to also output to other directories. But I also can't help but wonder if we should support more than one output folder. 🤔

Reply:
Technically we should probably output to the same set of directories that typescript is emitting to, but we don't have a general solution for that in place.